src/lib/remote-place-cache.ts

```diff
@@ -118,4 +152,11 @@ export default class RemotePlaceCache {
 
     return [];
   }
+
+  // For testing purposes
+  public static hasData(domain: string, placeType: string): boolean {
```

Comment:
you can just access private members in tests. shouldn't this be a function within the spec.ts file?